### PR TITLE
Data API stuff

### PR DESCRIPTION
**Pull request by mikeshardmind**
While this would be reasonably safe to merge prior to Cog-Creators/Red-DiscordBot#4045 , that PR is still subject to change.

This PR serves as both the intended way of initially handling this for these cogs, as well as providing an example of what cog creators need to handle, and the sorts of editorial decisions which may apply.

### CODE REVIEW

File: modnotes/modnotes.py [hunk 1]
```diff
@@ -53,7 +53,13 @@ class ModNotes(commands.Cog):
...
     """
 
-    __version__ = "330.0.2"
+    __version__ = "339.1.0"
+
+    __end_user_data_statement__ = (
+        "This cog stores data provided to it for "
+        "the purpose of a permanent moderation note system. "
+        "\nThis cog does not respect the data APIs."
+    )
```
Comment (DiscordLiz) [inline modnotes/modnotes.py]: Just move this to the private repo so that people don't use this.

**Review by DiscordLiz — CHANGES_REQUESTED** [review]
Let's talk about this more later. We have permission from each user to keep logs and data after account deletion already, so if any of this is too much, don't bother with it.

File: modnotes/modnotes.py (hunk 1 above)
Comment (mikeshardmind) [reply modnotes/modnotes.py]: That would be the easier option here. I'm fine marking it as hidden and eventually migrating this to a schema which can tolerate anonymization though.

**Comment by DiscordLiz** [comment]
you need to update this for your changes to the design.

**Pull request merged**